Description: use older Nodejs module terser 4.x
 This partly reverses upstream git commit 9f7cbad.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2020-12-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/__snapshots__/test.js.snap
+++ b/test/__snapshots__/test.js.snap
@@ -9,6 +9,7 @@
     "exports": Array [],
     "fileName": "chunk-1.js",
     "implicitlyLoadedBefore": Array [],
+    "importedBindings": Object {},
     "imports": Array [],
     "isDynamicEntry": false,
     "isEntry": true,
@@ -25,6 +26,7 @@
     "exports": Array [],
     "fileName": "chunk-2.js",
     "implicitlyLoadedBefore": Array [],
+    "importedBindings": Object {},
     "imports": Array [],
     "isDynamicEntry": false,
     "isEntry": true,
